R/utils.R

```diff
@@ -156,3 +156,8 @@ mark_end <- function() {
 expect_no_message <- function(object, regexp = NA, ...) {
   testthat::expect_message(object = object, regexp = regexp, ...)
 }
+
+get_build_version <- function(env_var = "$build_version") {
+  build_version <- system2("echo", args = env_var, stdout = TRUE)
```

Comment:
is there a reason to not use `Sys.getenv()` here?

Reply:
because as I understood, `Sys.getenv()` returns R environment variables, but not system variable, right?
```r
Sys.getenv("$HOME")
#> [1] ""
system2("echo", args = "$HOME", stdout = TRUE)
#> [1] "/Users/cj2dii"
```

to be honest, that's always been a bit confusing to me, so I could be not getting it right

Reply:
Don't include the `$`.
```r
R> Sys.getenv("HOME")
[1] "/Users/alexaxthelm"
R> Sys.getenv("$HOME")
[1] ""
R> system2("echo", args = "$HOME", stdout = TRUE)
[1] "/Users/alexaxthelm"
```

Reply:
changed to use `Sys.getenv()`... if that works for you, let's merge

Reply:
I restarted tests.... first round failed, but it was on the checkout of pacta-data repo, so should be not related to this change

Reply:
2nd round of tests passed